start.html

```diff
@@ -178,145 +181,341 @@ <h3 class="accordion">
+                           <li>Tell ansible where to find these roles.</li>
+                           <li>Create a playbook (you can copy the one below).</li>
+                           <li>Create an inventory file naming the hosts on which you wish to install
+                             <span class="c_snip_inline">pbench-agent</span> and including information about where the config file and ssh key
+                             file can be found.</li>
+                           <li>Run the playbook.</li>
+                       </div>
+                       <p class="info_text"  style="text-align:justify">
+                         The playbook will install the pbench repo and it will then use it to
+                         install the pbench-agent RPM; finally, it will install the
+                         localization pieces (config file and ssh key) that will allow you to
+                         send results to the server.
+                       </p>
+                       <p class="info_text"  style="text-align:justify">
+                         The prerequisite assumption is that you can ssh as root without a
+                         password into the relevant set of hosts:
+                       </p>
+		       <p class="c_snip">
+                         ssh-copy-id root@host
+                         ...
+                       </p>
+                       <p class="info_text"  style="text-align:justify">
+                         for each host in your inventory (that includes <span class="c_snip_inline">localhost</span> if you are going to
+                         install <span class="c_snip_inline">pbench-agent</span> on your local host).
+                       </p>
+                     </div>
+                   </div>
+                   <!-- Subsection: Install ansible and ansible galaxy (one-time task)-->
+		   <div class="accordionCard">
+		     <h4 class="accordion">
+		       <button class="accordionBtn pf-c-accordion__toggle" aria-expanded="false">
+			 <i class="arrowIcon fas fa-angle-right pf-c-accordion__toggle-icon" aria-hidden="true"></i>
+			 <span class="pf-c-accordion__toggle-text">Install ansible and ansible galaxy (one-time task)</span>
+		       </button>
+		     </h4>
 		     <div class="explanation pf-c-accordion__expanded-content pf-m-fixed" hidden>
 		       <div class="pf-c-accordion__expanded-content-body">
 			 <p class="info_text"  style="text-align:justify">
-			   You will need to install both the external
-			   repo and (if applicable) the internal repo
-			   on every machine where you are planning to
-			   run Pbench (including VMs). N.B. These
-			   steps must be run as root.
-			 </p>
-			 <p class="info_text"  style="text-align:justify"><b>External repo : </b></p>
+                           That is done using the standard package installer (<span class="c_snip_inline">dnf</span> or <span class="c_snip_inline">yum</span>):
```

Comment:
Perhaps just add that `yum` is still used on RHEL 7 environments.

Reply:
Fixed.